### ISSUE TITLE
Dota 2 Profile LIFETIME STATS removed after compendium update

### ISSUE THREAD

**Issue by beinelemental** [ValveSoftware/Dota2-Gameplay#12258]
### Description

I can't seems to find the LIFETIME STATS anymore anywhere in the widget selection. Is it removed intentionally or did they forgot to put it back or is it depends on your profile showcase budget ? Love the new profile showcase, just hope they add more stuff in the widget.

### Screenshots

![Capture](https://github.com/ValveSoftware/Dota2-Gameplay/assets/106768338/453c5977-1cfd-4801-a244-4decb05ce9f3)

**Comment by martymartz** [comment]
yes this is a terrible loss idk why they did this dont they know that stuff and the most successful heroes was very something players really liked ?

**Comment by martymartz** [comment]
this is missing and its not good, shame on valve

**Comment by eXoticD** [comment]
+1
please add this back !!

**Comment by martymartz** [comment]
+1
